### PR TITLE
MI additional form updates

### PR DESCRIPTION
**Pull request by nexussays**
https://app.asana.com/0/1186780595903796/1197113676435473

### CODE REVIEW

**Comment by nexussays** [comment]
This can merge if approved in https://app.asana.com/0/1186780595903796/1198176573006073

File: power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+   ( { choices, inputName } ) => (
+      <select name={inputName} required>
+         <option value=""></option>
+         {choices.map( x => <option value={x}>{x}</option> )}
+      </select>
+   );
+
+/**
+ * Michigan requires additional info be collected.
+ * @see
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx]: Does this link to anything?

File: power-the-polls-form/src/components/ptp-info-jurisdiction/ptp-info-jurisdiction.tsx [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
+   ( { state, stateInfo, jurisdictionInfo } ) => (
+      <Fragment>
+         <div class="next-steps">
+            {( // see: https://docs.google.com/document/d/10ngLtEP5wv48aNry3OzCgFhmzguBoSPNJtQfRS4Xn8Y/edit
+               state === "MI" ? [
+                  () => <Fragment>
+                     <strong>You'll hear from a partner in the next week</strong> about how you can help serve as a poll worker in Michigan.
+                           </Fragment>,
+                  () => "In the meantime, learn more about hours, compensation, and requirements for your community below and encourage your friends and family to sign up to be poll workers and help ensure a safe and fair election!",
+               ] : ( jurisdictionInfo?.application == null || jurisdictionInfo?.application === "" ) && stateInfo && stateInfo.usePhoneInsteadOfEmailForFormFallback ? [
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/ptp-info-jurisdiction.tsx]: you could simplify this to be !! jurisdictionInfo?.application ===  false

File: scripts/Readme.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Infrastructure Scripts
+
+## enrich-actionkit-data
```
Comment (Samcbehrens) [inline scripts/Readme.md:3]: <3

File: power-the-polls-form/src/components/ptp-info-poll-worker/ptp-info-poll-worker.tsx [hunk 4]
```diff
@@ -46,19 +46,15 @@ export class PollWorkerInfo {
       const { state, county, city } = this;
       const jurisdictionId = state ? findJurisdictionId( state, county, city ) : null;
 
-      return jurisdictionId != null ?
+      return jurisdictionId != null || state === "MI" ?
          (
             <ptp-info-jurisdiction
-               jurisdictionId={jurisdictionId}
+               jurisdictionId={jurisdictionId || -1}
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-poll-worker/ptp-info-poll-worker.tsx:52]: why the -1?

File: power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx (hunk 1 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx]: I do not know...

Comment (Samcbehrens) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx]: what does @see suppose to do ?

File: power-the-polls-form/src/components/ptp-info-poll-worker/ptp-info-poll-worker.tsx (hunk 4 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-poll-worker/ptp-info-poll-worker.tsx:52]: We know there is a hardcoded check for MI in ptp-info-jurisdiction, so I'd rather pass a number that will never be used vs retyping `jurisdictionId` to allow null and having to account for that in ptp-info-jurisdiction.

File: power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx (hunk 1 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx]: Reference another file or URL. I was probably going to, or used to, reference the component where this was used.

**Review by Samcbehrens — APPROVED** [review]

**Pull request merged**